playbooks/tasks/scs_compliance.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Download, install, configure, and execute SCS KaaS compliance check
+  vars:
+    check_dir: "{{ ansible_user_dir }}/scs-compliance"
+    python_venv_dir: "{{ ansible_user_dir }}/scs-compliance/venv"
+  block:
+  - name: Check if `kubeconfig_path` variable is defined
+    ansible.builtin.fail:
+      msg: "kubeconfig_path is not defined or empty"
+    when: kubeconfig_path is not defined or kubeconfig_path == ''
+  - name: Ensure check directory
+    ansible.builtin.file:
+      path: "{{ check_dir }}"
+      state: directory
+      mode: 0755
+  - name: Get SCS KaaS compliance check assets
+    ansible.builtin.git:
+      repo: https://github.com/SovereignCloudStack/standards.git
+      dest: "{{ check_dir }}"
+      single_branch: true
+      version: main
+  - name: Install virtualenv
+    ansible.builtin.package:
+      name: virtualenv
+    become: true
+  - name: Install check requirements
+    ansible.builtin.pip:
+      requirements: "{{ check_dir }}/Tests/requirements.txt"
+      virtualenv: "{{ python_venv_dir }}"
+  # TODO: We are simply copying a random check config template here.
+  #  A better solution would be to provide a configuration file that will serve as the source for all SCS checks.
+  #  This issue should be addressed in the scs/standard repository.
+  - name: Copy SCS KaaS compliance config files
+    ansible.builtin.copy:
+      src: "{{ check_dir }}/Tests/kaas/k8s-version-policy/config.yaml.template"
+      dest: "{{ check_dir }}/Tests/config.yaml"
+      remote_src: true
+      mode: 0644
```

Comment:
This task can (and should) be dropped.

Reply:
Then I believe that the new version of scripts won't complain that they cannot find the ./config.yaml. Is that true?

Reply:
Yes, that is true! We are still working on the [pull request](https://github.com/SovereignCloudStack/standards/pull/508), but so much is clear already.

Reply:
99c00d842db998107e329187d0c50ea236b3b7fb